File: Aula03/ListaPessoa/Nodo.cs
using System;

namespace Aula03_ListaPessoa
{    
    class Nodo {
        private Pessoa pessoa;
        private Nodo proximo;
        
        public Nodo(Pessoa pessoa1) {
            this.pessoa = pessoa1;
            this.proximo = null;
        }

        public Pessoa getPessoa() {
            return pessoa;
        }

        public void setPessoa(Pessoa pessoa) {
            this.pessoa = pessoa;
        }

        public Nodo getProximo() {
            return proximo;
        }

        public void setProximo(Nodo proximo) {
            this.proximo = proximo;
        }
    }
    
}

File: Aula06/Pilha/PilhaJava/Nodo.java

public class Nodo
{
    private String nome;
    private Nodo proximo;


    public Nodo(String nome1)
    {
        this.nome = nome1;
        this.proximo = null;

    }

    public Nodo()
    {

    }

    public String getNome()
    {
        return nome;
    }

    public void setNome(String nome1)
    {
        this.nome = nome1;
    }

    public Nodo getProximo()
    {
        return proximo;
    }

    public void setProximo(Nodo proximo)
    {
        this.proximo = proximo;
    }

}
